tests/python/test_ops.py

```diff
@@ -286,3 +286,198 @@ def test_errors(op: OpInfo, dtype: torch.dtype):
             exception_type, match=_regex_escape_parenthesis(exception_regex)
         ):
             errors_test_fn(op, sample)
+
+
+@pytest.mark.skip("issue 3740")
```

Comment:
Please add the issue link as a comment: https://github.com/NVIDIA/Fuser/issues/3740. Usually easier for navigating around.